src/script/common/c_content.cpp

```diff
@@ -128,6 +128,77 @@ void push_item_definition(lua_State *L, const ItemDefinition &i)
 	lua_setfield(L, -2, "description");
 }
 
+void push_item_definition_full(lua_State *L, const ItemDefinition &i)
+{
+	std::string type;
+	// This code is pure garbage, but that is what reviewers are for!
```

Comment:
We're not just here to tell you that you can't write good code. Look up the code style guidelines or some newer created code parts to see how it should be done.